CHANGES/6913.misc

```diff
@@ -0,0 +1 @@
+Allow one to specify a `pulp_ld_library_path` when wanted
```

Comment:
Since this is a net-new feature for the installer I think this should be `CHANGES/6913.feature` and the redmine issue https://pulp.plan.io/issues/6913 changed to a feature.

Reply:
Also @Spredzy thank you for the PR!